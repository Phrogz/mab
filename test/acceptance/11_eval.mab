p( "Test#11" )
setSlot( "makeExpression", method( x,
	Expression new( x )
) )
setSlot( "x", 42 )
setSlot( "myExp", makeExpression( 17 ) )
p( myExp eval( Lawn ), myExp eval )