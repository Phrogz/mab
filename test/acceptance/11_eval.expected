Test#11
42
17